### ISSUE TITLE
Deployment claims to be successful, while in fact still building

### ISSUE THREAD

**Issue by hirako2000** [vercel/vercel#730]
It is sort of a weird setup, but my `now-build` executes the server and binds to a port.
Once there, `now` claims deployment is successful
```
1. Synced   59s
2. Prepared container  8s
3. Installed and built project  48s
4. Started!   52m
READY! VISIT NOW 
```
But in fact it's still in the `build` phase, and will time out after 15mins, causing `now` to then rebuild if the instance is accessed.

instance log:
```
08/10 07:38 PM (10m)
npm run now-build
08/10 07:38 PM (10m)

> gisteam@0.9.0 now-build /home/nowuser/src
> gulp build && cross-env NODE_ENV=production node .
08/10 07:38 PM (10m)

[12:38:25] Using gulpfile ~/src/gulpfile.js
08/10 07:38 PM (10m)
[12:38:25] Starting 'build'...
08/10 07:38 PM (10m)
[12:38:25] Starting 'clean'...
08/10 07:38 PM (10m)
[12:38:25] Finished 'clean' after 12 ms
08/10 07:38 PM (10m)
[12:38:25] Starting 'copy'...
08/10 07:38 PM (10m)
[12:38:25] Finished 'copy' after 144 ms
08/10 07:38 PM (10m)
[12:38:25] Starting 'babel'...
08/10 07:38 PM (10m)
[12:38:26] Finished 'babel' after 1.28 s
08/10 07:38 PM (10m)
[12:38:26] Starting 'gzip'...
08/10 07:38 PM (10m)
[12:38:26] Finished 'gzip' after 41 ms
08/10 07:38 PM (10m)
[12:38:26] Finished 'build' after 1.49 s
08/10 07:38 PM (10m)
Thu, 10 Aug 2017 12:38:29 GMT koa deprecated Support for generators will be removed in v3. See the documentation for examples of how to convert old middleware https://github.com/koajs/koa/blob/master/docs/migration.md at lib/server.js:94:5
08/10 07:38 PM (10m)
info: Server now listening on 3000
```
https://zeit.co/hirako2000/gisteam/wlrojcrklj/logs

The status should probably says it's still building. 
It is also a problem `now-build` can be hijacked to actually start an application. with the current behaviour, this causes multiple re-build per day, in a loop, every 15mins if the instance is accessed.

**Comment by OlliV** [comment]
I don't think there is nothing we can do to help this.

**Comment by hirako2000** [comment]
Yes, there is something that can be done: Close the issue.

As the OP I have to say `now` support has been very communicative, the explanation given is fair enough, as it has to do with scalability and the concept of stateless servers.

However, it would be better if this issue was addressed. My case was down to some caching done by the server, so I had to make a choice between giving up on marko.js (unless hacking it) or on `now`.
This issue caused me some headache as I was troubleshooting deployment rather than realising that my app writing to disk was the source of the problem.

If caching to disk is not tolerated by `now`, I would say it should at least throw a warning when the app is attempting to do so, or even terminate the app altogether with a fatal error. That way we would know that disk writing post-deployment is not granted.
